doofinder-for-woocommerce/includes/api/class-reset-credentials-api.php

```diff
@@ -58,7 +58,7 @@ public function __construct($language)
         );
 
         $this->log->log('Create Management API Client');
-        $this->log->log('API Key: ' . $this->api_key);
+        // $this->log->log('API Key: ' . $this->api_key);
```

Comment:
Why commented and not deleted?

Reply:
My idea was to preserve it just in case we would like to uncomment it in specific debugging cases, but it can be also deleted :thinking: Ok, I'm deleting them

Reply:
Changed on: https://github.com/doofinder/doofinder-woocommerce/pull/262/commits/7c61b4dbce4593ffdb33b91054b39fc708ad3e94